tests/integration/integration-test.py

```diff
@@ -61,14 +61,15 @@ def test_get_pets_pagination(self, endpoint='/pets'):
         for pagination in testing_paginations:
             params = {f'page[{k}]': pagination[k] for k in ['number', 'size']}
             expected_status_code = pagination['expected_status_code']
+            resource = 'PetResource' if expected_status_code == 200 else \
+                'Error'
```

Comment:
Minor comment. Here is a post about how to make a line break on the ternary operation if line goes too long: https://stackoverflow.com/questions/28897010/how-to-make-a-line-break-on-the-python-ternary-operator